### ISSUE TITLE
Revamp the http://maemo.org homepage to promte Maemo Leste

### ISSUE THREAD

**Issue by dderby** [maemo-leste/bugtracker#147]
The community owns the Maemo name, trademark, domain, etc.  Leste is going to be an official Maemo release, therefore we should redesign and revamp the homepage to promote it.  Right now it presents years old Announcements, inactive community members under People and displays an empty News section.  To a casual visitor, it would probably look like Maemo has been completely abandoned when of course this in fact is not the case.

**Comment by MerlijnWajer** [comment]
It would be nice to also fixup maemo-leste.github.io to at least have the device pages moved over to our wiki before we do that, I think.

**Comment by sicelo** [comment]
For maemo.org, I guess you/we would need to contact the Maemo Council with the idea.
Of course there are also Maemo people who are not really interested in Leste, and who may not exactly be happy to see it displayed prominently in that way. For those people, "Leste is going to be an official Maemo release" == FALSE.
Either way, Council will evaluate it without bias.

**Comment by MerlijnWajer** [comment]
Yeah, to be fair, I'm OK with the current thread. Other people are also linking to us -- wikipedia, debian, devuan, etc. Let's not push for this until we're further along. At least nothing that calls it an 'official' release. I don't want to deal with haters - prefer they just stay away, so let's try to achieve that.

**Comment by dderby** [comment]
@sicelo, does the Maemo Council still exist?  They haven't posted any updates in ages, council terms are normally 6 months and the last election was over a year ago.  It seems like it's been abandoned.  Who should we contact?  Anyone know who's in charge of running the website?

**Comment by sicelo** [comment]
It does exist, but has been largely inactive for a while. However, I contacted one of the members a few weeks ago, and it seems that we might see some activity early next year.

**Comment by dderby** [comment]
@sicelo, what's the latest on this?  Another year on and I don't think we've heard a peep from the [last council](https://wiki.maemo.org/Community_Council).

@juiceme, @eekkelund, @eLtMosen, any thoughts on this?

**Comment by juiceme** [comment]
Yes, there has not been much activity lately.

I tried to rouse people last year to get up and hold at least the GA meeting so we'd get that sorted out as court is hazzling us to do the annual registration of Maemo e.V.

Similarily we'd need to try to get new council voting started, or else nothing will happen I think.

Regarding the maemo.org website revamping; yes that can be done but of course it first requires that somebody defines tha changes and what it should look like. Do that and I can then make the actual changes. :)

**Comment by dderby** [comment]
Thanks for replying @juiceme!  Good to hear that there are possibilities to revive the council and I'm pleased that you are supportive of a website revamp.

As for what it should look like; well I opened this issue so that we could discuss that. :)

My initial thoughts are that Maemo is alive and relevant, and that's the first thing we need to show.  There are all sorts of problems with the current homepage as I described in the first post but perhaps some of it is at least useful to Fremantle users.  The Maemo Leste homepage is currently https://maemo-leste.github.io/ which maybe we can use as the maemo.org homepage for now and then add a link back to the old homepage in case it's still needed.  That might be a good starting point.

I'm not a web designer so I'm not really the best person to ask on how it should look like, so it'd be good if others could chime in with suggestions.

**Comment by dderby** [comment]
I'd also recommend making it HTTPS only.  See here for reasons why we shouldn't be using HTTP for static content: https://www.troyhunt.com/heres-why-your-static-website-needs-https/

**Comment by MerlijnWajer** [comment]
I'm not so sure "replacing" the main page with our github page makes sense. I actually use it on an almost daily basis if only to look through packages. But some updated text, stuff like that seems useful for sure.

**Comment by MerlijnWajer** [comment]
It's not (even) listed here ATM http://wiki.maemo.org/Projects

**Comment by dderby** [comment]
> I'm not so sure "replacing" the main page with our github page makes sense. I actually use it on an almost daily basis if only to look through packages. But some updated text, stuff like that seems useful for sure.

Maybe I wasn't very clear .  What I mean is do something like `mv index.html old_index.html`, then if you replace the main page with the github page, you can add a link to the new page that points to old_index.html.  That way you'll still be able to use it to look through packages.

**Comment by freemangordon** [comment]
I agree with Merlijn that replacing the old well-known meamo.org homepage with the one from github.io is not something we want to do, at least as of now.

**Comment by dderby** [comment]
Understood.  Let's see if someone steps up to help with a redesign.

**Comment by pavelmachek** [comment]
https topic... It would be nice if maemo.org webpage could be accessed from Maemo phones... but they have old browsers and have problems with many https websites ("no common ciphers").
